src/lib/sidenav/sidenav.ts

```diff
@@ -316,6 +317,35 @@ export class MdSidenavContainer implements AfterContentInit {
   /** The sidenav child with the `end` alignment. */
   get end() { return this._end; }
 
+  /**
+   * Width of the container at which it breaks
+   * e.g., breakpointWidth="500"
+   */
+  @Input() breakpointWidth: number;
+
+  /** Mode of the breakpoint; either true or false */
+  @Input() breakpointChangeMode: true | false = true;
```

Comment:
Try this:

```ts
/** Whether the the drawer changes modes when collapsing. */
```

Note: the terminology changes from sidenav to drawer in a pretty recent commit